pkgs/top-level/all-packages.nix

```diff
@@ -3580,6 +3580,8 @@ with pkgs;
 
   odafileconverter = libsForQt5.callPackage ../applications/graphics/odafileconverter {};
 
+  oil-buku = callPackage ../applications/misc/oil { };
```

Comment:
This should probably match the attr name

```
  oil-buku = callPackage ../applications/misc/oil-buku { };

Reply:
ok, will do